tool/since.dart

```diff
@@ -11,14 +11,18 @@ import 'package:yaml/yaml.dart';
 
 import 'crawl.dart';
 
+// Flip to (re)generate since/linter.yaml contents.
+var regenerateLinterYaml = false;
+
 void main() async {
-// Uncomment to (re)generate since/linter.yaml contents.
-//  for (var lint in registeredLints) {
-//    var since = await findSinceLinter(lint.name);
-//    if (since != null) {
-//      print('${lint.name}: $since');
-//    }
-//  }
+  if (regenerateLinterYaml) {
+    for (var lint in registeredLints) {
+      var since = await findSinceLinter(lint.name);
+      if (since != null) {
+        print('${lint.name}: $since');
+      }
+    }
+  }
 
 // Uncomment to (re)generate since/dart_sdk.yaml contents.
 //  var tags = await sdkTags;
```

Comment:
I wanted to uncomment this code, and guard with a bool, like above. But it has completely bitrotted. It references fields or functions which do not exist. Delete?

Reply:
@pq what about here? If I were tasked with regenerating `since/dart_sdk.yaml`, and I uncommented this code and found it was broken, I wouldn't really know where to start. If it had instead been not commented, it would not have compile-time errors.